release-management.md

```diff
@@ -14,31 +14,32 @@ The maintainers may periodically update this policy based on feedback.
 
 ## Release Versioning
 
-Consumers of the go-cose project may build directly from main, or pull from released builds.
-Builds from main must reference the git-commit as the version: `v1.0.0-2300d5c`
```

Comment:
The `git-commit` is not SemVer2 valid since `v1.0.0-2300d5c` < `v1.0.0-alpha.1` < `v1.0.0-b300d5c`.